cocotb/share/makefiles/Makefile.sim

```diff
@@ -136,11 +136,7 @@ define check_for_results_file
     @test -f $(COCOTB_RESULTS_FILE) || (echo "ERROR: $(COCOTB_RESULTS_FILE) was not written by the simulation!" >&2 && exit 1)
 endef
 
-# Append directory containing the cocotb Python package to PYTHONPATH
-# XXX: This is only needed if cocotb is not installed into the default
-# Python search path.
-PYTHONPATH := $(COCOTB_PY_DIR):$(PYTHONPATH)
-export PYTHONPATH
+PYTHONPATH := $(PYTHONPATH):$(PWD)
```

Comment:
Any idea why we need `PWD`?

Reply:
It is assumed that in user makefiles. If not would need to add this in every makefile or add to `sys.path` example in `__init__.py`

Reply:
What puzzles me is how when I run `python` normally, `sys.path` automatically contains `.`.

Is this not the default for embedded python?

Reply:
This is strange but this is mention in multiple places (I can google for you). Founded it by chance.

Reply:
Seems like. Could add to `sys.path` just before: https://github.com/cocotb/cocotb/blob/master/cocotb/regression.py#L151

Reply:
Let's put off this converation for now, this isn't actually a change made by this patch.